browser/src/Services/Configuration/DefaultConfiguration.ts

```diff
@@ -213,6 +213,100 @@ const BaseConfiguration: IConfigurationValues = {
             "TypeScriptReact.tmLanguage.json",
         ),
     },
+    "language.lua.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "lua",
+        "syntaxes",
+        "lua.tmLanguage.json",
+    ),
+    "language.clojure.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "clojure",
+        "syntaxes",
+        "clojure.tmLanguage.json",
+    ),
+    "language.ruby.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "ruby",
+        "syntaxes",
+        "ruby.tmLanguage.json",
+    ),
+    "language.swift.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "swift",
+        "syntaxes",
+        "swift.tmLanguage.json",
+    ),
+    "language.rust.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "rust",
+        "syntaxes",
+        "rust.tmLanguage.json",
+    ),
+    "language.php.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "php",
+        "syntaxes",
+        "php.tmLanguage.json",
+    ),
+    "language.objc.textMateGrammar": {
+        ".m": path.join(
+            __dirname,
+            "extensions",
+            "objective-c",
+            "syntaxes",
+            "objective-c.tmLanguage.json",
+        ),
+        ".h": path.join(
+            __dirname,
+            "extensions",
+            "objective-c",
+            "syntaxes",
+            "objective-c.tmLanguage.json",
+        ),
+    },
+    "language.objcpp.textMateGrammar": {
+        ".mm": path.join(
+            __dirname,
+            "extensions",
+            "objective-c++",
+            "syntaxes",
+            "objective-c++.tmLanguage.json",
+        ),
+    },
+    "language.python.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "python",
+        "syntaxes",
+        "python.tmLanguage.json",
+    ),
+    "language.sh.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "shell",
+        "syntaxes",
+        "shell.tmLanguage.json",
+    ),
+    "language.zsh.textMateGrammar": path.join(
+        __dirname,
+        "extensions",
+        "shell",
+        "syntaxes",
+        "shell.tmLanguage.json",
+    ),
+    "language.markdown.textMateGrammar": {
+        ".md": path.join(__dirname, "extensions", "rust", "syntaxes", "rust.tmLanguage.json"),
+        ".markdown": path.join(__dirname, "extensions", "rust", "syntaxes", "rust.tmLanguage.json"),
```

Comment:
These seem to be pointing at the rust grammars at the moment.

Reply:
Thanks for catching that 👍 , it was fairly mindless work, tested them in some buffers and did note that the `md` ones were fairly unremarkable, guess that explains why 😆